### ISSUE TITLE
Plans on releasing a new python package

### ISSUE THREAD

**Issue by plainerman** [e3nn/e3nn#379]
It has been quite some time since the last python release (v0.5.0). In the meantime you fixed a few things such as the batchnorm for newer pytorch versions: 2edaa269c1ca82b93f5bb78614074bf76825f0ea. Our team is currently facing problems with the current upstream implementations and we were wondering whether you plan on releasing a new version soon or if we should install the dependency directly from source?

Especially since it says "e3nn is under development. It is recommanded to install using pip. The main branch is considered as unstable." in the readme, we would like to avoid that.

**Comment by mariogeiger** [comment]
Indeed, that was a while ago... I'm mostly working on the jax version.

**Comment by plainerman** [comment]
Thank you very much! 🥳
